app/routes/users/components/UserSettings.js

```diff
@@ -51,11 +52,7 @@ const UserSettings = (props: Props) => {
   const showAbakusMembership = user.isStudent;
 
   const onSubmit = data =>
-    updateUser(omit(data, 'profilePicture')).catch(err => {
-      if (err.payload && err.payload.response) {
-        throw new SubmissionError(err.payload.response.jsonData);
-      }
-    });
+    updateUser(omit(data, 'profilePicture')).catch(handleSubmissionError);
```

Comment:
This `omit` can be removed when we introduce "picking" in `legoForm`?? Since there is no `profilePicture` field..